qa-tests-backend/src/test/groovy/ExternalNetworkSourcesTest.groovy

```diff
@@ -44,7 +42,19 @@ class ExternalNetworkSourcesTest extends BaseSpecification {
         return deployment
     }
 
+    @Shared
+    private int mask = 0
+
+    private String getBigCIDR() {
+        return "$CF_IP_ADDRESS/${20+mask}"
+    }
+
+    private String getSmallCIDR() {
+        return "$CF_IP_ADDRESS/${30-mask}"
+    }
```

Comment:
There is a danger that those would collide when `mask==5`

Reply:
Can we maybe do `mask = mask % 5` to prevent that to some degree?

Reply:
We can extend the range e.g. start from 15 instead of 20